Makefile

```diff
@@ -61,6 +61,14 @@ al2gpu: check-region init validate release.auto.pkrvars.hcl
 al2inf: check-region init validate release.auto.pkrvars.hcl
 	./packer build -only="amazon-ebs.al2inf" -var "region=${REGION}" .
 
+.PHONY: al2kernel5dot10
+al2kernel5dot10: check-region init validate release.auto.pkrvars.hcl
```

Comment:
its more like a AL2 5.x AMI right? i.e. the minor version (.10) may keep changing? 

if so ^^, then maybe the `dot10` may not be suited. how about something like `al2kernel5x`?

Reply:
I see Mythri asking the same q above

Reply:
@mythri-garaga @singholt thanks for the comments! Product has indicated we will not be introducing any minor version updates to the kernel for AL2 after adding support for kernel 5.10. The long term plan is to help customers move to [AL2022, which is planned to be GA with kernel version at least 5.15 with 5 years of support](https://github.com/amazonlinux/amazon-linux-2022/issues/4#issuecomment-1313874947).